go/vt/vtgate/executor.go

```diff
@@ -156,6 +156,7 @@ func (e *Executor) Execute(ctx context.Context, method string, safeSession *Safe
 	saveSessionStats(safeSession, stmtType, result, err)
 	if result != nil && len(result.Rows) > *warnMemoryRows {
 		warnings.Add("ResultsExceeded", 1)
+		log.Warningf("%q exceeds warning threshold of max memory rows: %v", sql, *warnMemoryRows)
```

Comment:
I think we may want to truncate the logged query: https://github.com/vitessio/vitess/blob/fe6d095ccae0f50356d4691b0171bbeb8778e911/go/vt/sqlparser/truncate_query.go#L50

When logged, long queries can create problems for log parsing programs / disk. See: https://github.com/vitessio/vitess/pull/6959

Reply:
cc @systay